pkg/storaged/storage-controls.jsx

```diff
@@ -227,7 +227,7 @@ export const StorageSize = ({ size }) => {
 };
 
 export const StorageMenuItem = ({ onClick, onlyNarrow, danger, excuse, children }) => (
-    <DropdownItem className={(onlyNarrow ? "show-only-when-narrow" : "") + (danger ? " delete-resource-dangerous" : "")}
+    <DropdownItem className={(onlyNarrow ? "show-only-when-narrow" : "") + (danger && !excuse ? " delete-resource-dangerous" : "")}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19898-20240129-124746-af56ad9b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)